### ISSUE TITLE
Thumbnails 

### ISSUE THREAD

**Issue by heyrobt** [getgrav/grav-premium-issues#57]
The thumbnails aren't appearing as I expected. Instead, they are in a vertical column. I read through the documentation several times and didn't see anything that might be helpful with this. I followed the example "Gallery Lightbox with Shortcodes", with the exception that I put the text description in the desc="" attribute. Everything works just fine, except the thumbnails are vertical. I don't know if this is an issue or that I'm overlooking something.

![756ABF6B-1420-4D1D-BEA2-AB612E0E85EA](https://user-images.githubusercontent.com/79666291/109399457-c4445580-78f7-11eb-9e86-cb7a00786c73.jpeg)

**Comment by rhukster** [comment]
If you are not using Typhoon theme, or another Tailwind-based theme, then you will need to add your own CSS as doesn't come with anything built-in.  I can whip up a quick example CSS and document that, but usually the theme you are using with have it's own CSS grid and styling classes that should be used.

**Comment by heyrobt** [comment]
That make sense. I'll watch for an example CSS and see how I can fit it in. I been using a slightly modified version of the Deliver theme.

**Comment by rhukster** [comment]
This should help: 

https://getgrav.org/premium/lightbox-gallery/docs#displaying-a-gallery-of-thumbnails

**Comment by kees-closed** [comment]
I use this as well, with the theme Quark: https://getgrav.org/premium/lightbox-gallery/docs#displaying-a-gallery-of-thumbnails-via-twig

And it works! Until I refresh right after saving any changes. First it looks like this:

<img width="870" height="845" alt="Image" src="https://github.com/user-attachments/assets/b6886df6-df24-4761-b90a-d8802eaf995b" />

And then after another refresh it becomes like this:

<img width="870" height="1075" alt="Image" src="https://github.com/user-attachments/assets/90482057-eaf0-4034-bb56-9b655c2c1f4f" />

I'm using the default theme Quark with Grav v1.7.52, Admin v1.10.51 and Lightbox v1.2.1 1.7 on PHP 8.3.31.

**Comment by rhukster** [comment]
So you are saying you are using a `gallery.md` file for your gallery view, and it works fine once, then after another reload the layout is broken?  can you disable cache and try that, does that fix your issue?

**Comment by kees-closed** [comment]
Hi @rhukster, yes I use `gallery:` in the Frontmatter, just like the documentation shows. And I included the `partials/gallery.html.twig` in my theme overlay for Quark.

Cache was indeed the issue. When I globally disable it, it loads fine. I enabled it again and only set this in the Frontmatter: `never_cache_twig: true` of the blog post. Now it works fine. So this workaround is fine for now, thanks! But a fix is always welcome of course :)
